src/WebJobs.Script.WebHost/Controllers/HostController.cs

```diff
@@ -156,6 +156,32 @@ public IActionResult DrainStatus([FromServices] IScriptHostManager scriptHostMan
             }
         }
 
+        [HttpPost]
+        [Route("admin/host/resume")]
+        [Authorize(Policy = PolicyNames.AdminAuthLevelOrInternal)]
+        public async Task<IActionResult> Resume([FromServices] IScriptHostManager scriptHostManager)
+        {
+            _logger.LogDebug($"Received request to resume a draining host - host status: {scriptHostManager?.State.ToString()}");
+
+            if (!Utility.TryGetHostService(scriptHostManager, out IDrainModeManager drainModeManager)
+                || scriptHostManager?.State != ScriptHostState.Running)
```

Comment:
Recommend checking state before accessing drain mode manager - can short circuit that check in many cases